app/server.R

```diff
@@ -453,21 +454,25 @@ server <- function(input, output, session) {
     if (input$scoreType == "wis" || input$scoreType == "sharpness") {
       # Only show WIS or Sharpness for forecasts that have all intervals unless they are for future dates
       filteredScoreDf <- filteredScoreDf %>%
-        filter(!(is.na(`50`) &&
-          is.na(`80`) &&
-          is.na(`95`) &&
-          target_end_date < dataCreationDate))
+        filter(!(
+          (is.na(`50`) |
+            is.na(`80`) |
+            is.na(`95`)
+          ) &
+            target_end_date < dataCreationDate))
```

Comment:
Confirming truth table because these do seem tricky:

| 50, 80, or 95 missing | target end date before data creation date | result |
| - | - | - |
| yes | yes | drop |
| yes | no | keep |
| no | yes | keep |

Reply:
Yes, that's right. We only want to "keep" when *all* of 50, 80, and 95 columns are available OR `target_end_date` >= `dataCreationDate` (prediction is in the future).

Is it clearer to write the "keep" criteria instead of the "drop" criteria? E.g.

```suggestion
        filter((
            !is.na(`50`) &
            !is.na(`80`) &
            !is.na(`95`)
          ) |
            target_end_date >= dataCreationDate)
```

Reply:
Keep criteria are clearer for me, but I'm not the one who'll be writing code here whose correctness depends on my boolean algebra reading comprehension 🙂 

@rafaelcatoia @qx-teo any opinion?

Reply:
I think either's fine? If we're afraid of clarity the easiest way would be just to comment, IMO